extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -80,7 +80,7 @@ static public function isCorrectPageType() {
 		}
 	}
 
-	static public function canShowDiscussions( $cityId ) {
+	static public function canShowDiscussions( $cityId, $ignoreWgEnableRecirculationDiscussions = false ) {
```

Comment:
this is not a hook, why this is placed in file called ReirculationHooks.class.php?